cores/esp8266/cont_util.c

```diff
@@ -57,7 +57,7 @@ int ICACHE_RAM_ATTR cont_get_free_stack(cont_t* cont) {
     return freeWords * 4;
 }
 
-bool ICACHE_RAM_ATTR cont_can_yield(cont_t* cont) {
+bool cont_can_yield(cont_t* cont) {
```

Comment:
This can be called from ISR, when cache is disabled, so it's best to keep it in IRAM.